lib/socket.c

```diff
@@ -883,13 +895,15 @@ connect_async_ai(struct smb2_context *smb2, const struct addrinfo *ai, int *fd_o
                 ((struct sockaddr_in *)&ss)->sin_len = socksize;
 #endif
                 break;
+#ifdef ENABLE_IPV6
```

Comment:
AF_INET6 should be available by default on most platforms, so instead of opting-in to have ipv6
can we change this to opt-out of ipv6 and only use it on those platforms that lack ipv6?

I.e. a #ifdef DISABLE_IPV6 instead

Reply:
On amiga os port looks more effective than Xbox 360 port.